src/internal/files/files_test.go

```diff
@@ -0,0 +1,76 @@
+package files
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFiles_WriteToJsonFile_Success(t *testing.T) {
```

Comment:
do we really need unit tests for a method that 100% just wraps standard lib?

I feel like we're not testing any of our code here, we're testing things that already exist and are maintained elsewhere.

Reply:
For me, it helped shape the error messages.  It's definitely a bit overkill, but IMO probably does not hurt?

Reply:
I think if we want to test the error messages then that's fine :D